tests/ProxyManagerTest/Functional/LazyLoadingGhostFunctionalTest.php

```diff
@@ -829,4 +829,67 @@ public function getPropertyAccessProxies()
             ],
         ];
     }
+
+    /**
+     * @dataProvider skipPropertiesFixture
+     *
+     * @param string   $className
+     * @param string   $propertyName
+     * @param string   $expected
+     * @param string[] $properties
+     */
+    public function testSkipProperties($className, $propertyName, $expected, $properties)
+    {
+        $factory     = new LazyLoadingGhostFactory();
```

Comment:
This test should not involve usage of the `LazyLoadingGhostFactory` - please see other tests in this class and how they get the proxy class